deeplearning4j/deeplearning4j-nlp-parent/deeplearning4j-nlp/src/main/java/org/deeplearning4j/iterator/BertIterator.java

```diff
@@ -341,6 +385,16 @@ public MultiDataSet next(int num) {
         return tokens;
     }
 
+
```

Comment:
Not really thrilled about this. But I couldn't think of another to be able to reuse code for inference.